src/components/widgets/filesystem/FileEditor.vue

```diff
@@ -52,10 +126,26 @@ export default class FileEditor extends Vue {
     rulers: (this.isMobile) ? [80, 120] : []
   }
 
+  structures: Structure[] = []
+
+  st: any[] = []
+
   get isMobile () {
     return this.$vuetify.breakpoint.mobile
   }
 
+  get configLink () {
+    const configMap = this.$store.getters['server/getConfigMapByFilename'](this.filename)
+    let link = configMap.link
+    if (configMap.service) {
+      const configLink = 'app.file_system.config.' + configMap.service + '.link'
+      if (this.$t(configLink)) {
```

Comment:
`$t()` always returns a string (it has a fallback to the string being input), so the wrong href is added to the helper links

```suggestion
      if (this.$t(configLink) !== configLink) {
```